src/azure-cli/azure/cli/command_modules/policyinsights/aaz/latest/policy_insights/attestation/_create.py

```diff
@@ -0,0 +1,360 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+class Create(AAZCommand):
+    """Create an attestation at resource scope.
+
+    :example: Create a minimal attestation
+        az policy attestation create --attestation-name 'Test-Attestation' -a '142412412412414'
+    """
+
+    _aaz_info = {
+        "version": "2022-09-01",
```

Comment:
Can we use constants here for the version number ?

Reply:
@najagasi Since that class is auto-generated, it will be updated next time we auto-generate if there is a new version